workflow/Snakefile

```diff
@@ -289,8 +289,8 @@ rule plot_nn_tc_ever_v_never:
     input:
         nn_tc_file = expand(f"{OUTPUT_DIR}/model_evaluation/{{enum_factor}}/{{dataset}}_{{repr}}_{{fold}}_nn_tc_ever_v_never.csv.gz",
             dataset=DATASET, repr=REPRESENTATIONS, fold=range(FOLDS), allow_missing=True),
-        ranks_file = expand(f"{OUTPUT_DIR}/{{enum_factor}}/prior/structural_prior/{{dataset}}_{{repr}}_min1_all_freq-avg_CV_ranks_structure.csv.gz",
-            dataset=DATASET, repr=REPRESENTATIONS, allow_missing=True)
+        ranks_file = expand(f"{OUTPUT_DIR}/{{enum_factor}}/prior/structural_prior/{{dataset}}_{{repr}}_{{fold}}_CV_ranks_structure.csv.gz",
```

Comment:
Initially, we were combining all fold-specific `nn_tc` never/ever outputs, and then comparing with `ranks_all` file. Now, we're merging one `nn_tc` file per fold with one `rank` file per fold.